lib/bin/check-file-headers.js

```diff
@@ -19,6 +19,8 @@ const header = `// Copyright 2017 ODK Central Developers
 // except according to the terms contained in the LICENSE file.
 `;
 
+const newHeader = header.replace('{{year}}', new Date().getFullYear());
```

Comment:
What do you think about keeping 2017 in the header, then replacing 2017 with the current year here?